_docs_integrate/01-integrate-scenarios.md

```diff
@@ -15,35 +15,39 @@ Here you'll find everything you need to seamlessly dive into the world of enmesh
 
-## Data Model
+## Get in touch with other Identities
+
+Learn how to [establish a Relationship to another Identity]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}).
 
-Understand enmeshed's data model and how it fits into your integration, including error codes and attribute values.
+## Working with Requests
 
-## Handling Events
+Learn how to create and manage Requests. Check out the [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}) and the [Requests over Templates]({% link _docs_integrate/requests-over-templates.md %}) and [Requests over Messages]({% link _docs_integrate/requests-over-messages.md %}) guides.
 
-Discover how events work and how you can leverage them in your application.
+## Manage Attributes of yourself
 
-## Manage Requests
+Explore how to [create Attributes for yourself]({% link _docs_integrate/create-attribute-for-yourself.md %}) and how to [share Attributes with a peer]({% link _docs_integrate/share-own-attribute-to-peer.md %}). Furthermore, learn how to [succeed Attributes to update their values]({% link _docs_integrate/succeeding-attributes-to-update-their-values.md %}).
 
-Learn how to create and manage requests.
+## Manage Attributes of others
 
-## Manage Attributes of Others
+Explore how to [query Attributes]({% link _docs_integrate/read-attribute-from-peer.md %}) and [store Attributes]({% link _docs_integrate/create-attribute-for-peer.md %}) in the Wallet for others.
 
-Explore how to store and query attributes in the Wallet for other users.
+## Basic communication
 
-## Messages
+Explore how to [send Messages]({% link _docs_integrate/sending-messages.md %}) through enmeshed for simple and secure communication.
 
-Explore how to send messages through enmeshed for simple and secure communication.
+## Request consent
 
-## Programming
+Discover how to request [persistent consent]({% link _docs_integrate/request-persistent-consent-of-peer.md %}) and [one-time consent]({% link _docs_integrate/requesting-one-time-consents.md %}) of a peer.
+
+## Data Model
 
-Gain insights into effectively utilizing the Connector SDKs to streamline and optimize your integration.
+Understand [enmeshed's data model]({% link _docs_integrate/data-model-overview.md %}) and how it fits into your integration. Also get an overview of the [Connector events]({% link _docs_integrate/connector-events.md %}), the [Attribute values]({% link _docs_integrate/attribute-values.md %}) and our [use cases]({% link _docs_integrate/use-cases.md %}).
```

Comment:
For me "the data model of enmeshed" sounds a bit smoother than "enmeshed's data model" (it took me some tries to pronounce this correctly in my head :D). But not too important here.